atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -293,12 +297,14 @@ static LockAndTimestampServices createRawServices(
             Environment env,
             Supplier<RemoteLockService> lock,
             Supplier<TimestampService> time,
+            Supplier<TimestampStoreInvalidator> invalidator,
```

Comment:
Right. I think I originally went with it for consistency, but you're right that for invalidator there's no need for this; removed to just pass `TimestampStoreInvalidator` around.

(Also for timestamp service we'd prefer a supplier since it fires off some logic on creation to set up the caching timestamp buffer. I've checked that creating a timestamp store invalidator doesn't have any such overheads.)